config.yaml

```diff
@@ -72,6 +72,13 @@ clients:
       blockwiseTransfer:
         enabled: true
         blockSize: "1024"
+      ownershipTransfer:
+        method: justWorks
```

Comment:
Does it make sense to configure it? If device supports just works, do just works. If it supports manufacturer certificate only, and it's configured, use it. If it's not configured, own will fail. What's the benefit of configuring what method to use, if it can be decided during OTM?

Reply:
This is about enforcing - you don't want to own the device via justworks and connect device to hub .

Reply:
Well, true.

Reply:
We can change it to array where you define which methods are supported

Reply:
Not supported, but allowed

Reply:
Fixed